src/js/administration/components/Server.js

```diff
@@ -1,15 +1,9 @@
 import React from "react";
 import { NarrowContainer } from "../../base";
-import Sentry from "./Sentry";
 import API from "./API";
 
 export const ServerSettings = () => (
-    <>
-        <NarrowContainer>
-            <Sentry />
-        </NarrowContainer>
-        <NarrowContainer>
-            <API />
-        </NarrowContainer>
-    </>
+    <NarrowContainer>
+        <API />
```

Comment:
Deepsource is complaining about this not being Pascal case. I looked it up and the recommendation is to use `Api` instead.

Can you update this?